doc/integrator/print.rst

```diff
@@ -0,0 +1,217 @@
+.. _integrator_print:
+
+Print
+=====
+
+Adding the *print* feature to a c2cgeoportal project involves adding
+a ``cgxp.plugins.Print`` plugin to the viewer, and configuring the MapFish
+Print.
+
+The Print plugin
+----------------
+
+The viewer should include a ``Print`` plugin for the *print* feature to
+be available in the user interface.
+
+See the `Print API doc
+<http://docs.camptocamp.net/cgxp/2.3/lib/plugins/Print.html>`_ for the
+list of options the plugin can receive.
+
+*The main viewer includes a Print plugin by default.*
+
+Using MapFish Print v3
+----------------------
+
+Migration from the v2
+~~~~~~~~~~~~~~~~~~~~~
+
+Start the migration by instantiating the default template:
+
+.. prompt:: bash
+
+   cd <project_root>
+   cp -r print ~
+   git rm -r print
+   SRID=-1 ./docker-run pcreate --interactive --scaffold c2cgeoportal_create --package-name <package> /tmp/<project>
+   ./docker-run pcreate --interactive --scaffold c2cgeoportal_update --package-name <package> /tmp/<project>
+   mv /tmp/<project>/print .
+   rm -rf /tmp/<project>
+
+Then in ``~/print`` you will have a backup of the old template, and in the ``print`` folder you will have the new print configuration.
+
+In the ``<package>.mk`` you should remove the ``PRINT_VERSION = 2`` (it is ``3`` by default).
+
+In the concerned viewer javascript files, in the ``cgxp_print`` plugin add the following lines:
+
+.. code:: javascript
+
+   version: 3,
+   encodeLayer: {},
+   encodeExternalLayer: {},
+   additionalAttributes: []
+
+Template configuration
+~~~~~~~~~~~~~~~~~~~~~~
+
+All print-related files are located in the ``print/`` folder, and the files related to the template in the
+``print/print-apps/<package>`` folder.
+
+The main configuration file is ``print/print-apps/<package>/config.yaml``.
+
+* `MapFish print documntation <http://mapfish.github.io/mapfish-print-doc/>`_
+* `Startup with Jasper Reports <http://mapfish.github.io/mapfish-print-doc/#/jasperReports>`_
+
+
+MapFish Print v2 configuration
```

Comment:
don't works on 2.3

Reply:
I will move this section to legacy_print.rst

Reply:
It's not legacy, it don't works ...

Reply:
OK, then I simply remove this part